### PR TITLE
Includes Docs at /docs

### PR DESCRIPTION
**Pull request by b-m-f**
## Done

Includes Docs to be shown at /docs. Header taken from #30.

## QA

    View the site at https://microk8s-io-canonical-websites-pr-31.run.demo.haus/docs/
    Make sure that the docs are presented as they should be.
 
## Issues
#26 
#27 
#28
#29

### CODE REVIEW

File: _layouts/base.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <!-- Google Tag Manager --
```
Comment (nottrobin) [inline _layouts/base.html]: You should re-enable the tag manager tags. Use the same one as the rest of the site.

File: _layouts/base.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+  <title>microk8s - Fast, Light, Upstream Developer Kubernetes</title>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" href="https://assets.ubuntu.com/v1/b3b72cb2-canonical-logo-166.png" type="image/x-icon" />
+  <link rel="stylesheet" href="/css/styles.css" />
+</head>
+
+<body>
+  <!-- Google Tag Manager (noscript) --
```
Comment (nottrobin) [inline _layouts/base.html]: As above

File: docs/dockerd.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+layout: base
```
Comment (nottrobin) [inline docs/dockerd.md]: For within this site, you should probably rename this layout to `docs`.

File: _layouts/base.html (hunk 1 above)
Comment (b-m-f) [reply _layouts/base.html]: Extracted them into includable fragments and used those

File: _layouts/docs.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
+<!doctype html>
+<html lang="en">
+
+<head>
```
[the remark this answers is not in the document]
Comment (b-m-f) [reply _layouts/docs.html]: Good idea. Ill check

File: _layouts/docs.html [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+  {% include analytics.html %}
+  <title>microk8s - Fast, Light, Upstream Developer Kubernetes</title>
```
Comment (nottrobin) [inline _layouts/docs.html]: We should probably make the `<title>` more specific - something like:

```suggestion

<title>
  {% if page.title %}
    {{ page.title }} | microk8s
  {% else %}
    microk8s - Fast, Light, Upstream Developer Kubernetes
  {% endif %}
</title>
```

(Practices could do with something like this too)

File: _includes/head.html [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<head>
+  {% include analytics.html %}
```
Comment (nottrobin) [inline _includes/head.html]: If this include is now shared, the contents of `analytics.html` could simply be moved inside here again, so devs don't have to walk one extra step to find it.

File: _layouts/base.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+  When you come to work on it in ernest, please remove this comment
+  and replace index.html with this file.
+-->
+
+<html lang="en">
+{% include head.html %}
+
+<body>
+  {% include analytics_no_script.html %}
```
Comment (nottrobin) [inline _layouts/base.html]: Same as above - this content can probably be placed back in here directly.

File: _layouts/base.html [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+<!--
+  This is a basic HTML file template to become the new homepage of the site.
+  Currently it simply includes Vanilla styles and not much else.
+
+  When you come to work on it in ernest, please remove this comment
+  and replace index.html with this file.
+-->
+
+<html lang="en">
+{% include head.html %}
```
Comment (nottrobin) [inline _layouts/base.html]: Actually, if `base.html` is shared and is the only place that calls `head.html`, there's no need for `head.html` either.

File: _layouts/base.html [hunk 9]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```
Comment (nottrobin) [inline _layouts/base.html]: I :heart: this structure, basing both layouts of `base.html`

File: _sass/_old_vanilla.scss [hunk 10]
```diff
@@ -0,0 +1,4805 @@
+.p-icon--minus, .p-icon--plus, .p-icon--expand, .p-icon--collapse, .p-icon--chevron, .p-icon--close, .p-icon--help, .p-icon--information, .p-icon--delete, .p-icon--error, .p-icon--warning, .p-icon--external-link, .p-icon--contextual-menu, .p-icon--code, .p-icon--menu, .p-icon--copy, .p-icon--search, .p-icon--success, .p-icon--share, .p-icon--user, .p-icon--question, .p-icon--spinner {
```
Comment (nottrobin) [inline _sass/_old_vanilla.scss]: Why do you need `old_vanilla`? I thought both the docs.vanillaframework.io styling and this website were based off vanilla-framework 1.8.0? Or at least that they'd be close enough to work...?

Comment (b-m-f) [reply _sass/_old_vanilla.scss]: This is for the sidebar. 
I needed the `.p-sidebar` class. 
@anthonydillon is there a better way to do this? Or wait until sth lands in Vanilla and then come back here and fix?

File: _layouts/base.html (hunk 9 above)
Comment (b-m-f) [reply _layouts/base.html]: :)

File: _layouts/docs.html (hunk 5 above)
Comment (b-m-f) [reply _layouts/docs.html]: Oho. I will try this out. Should work when using the Parsing information correctly in the markdown docs.

Comment (b-m-f) [reply _layouts/docs.html]: Works great!

File: _sass/_old_vanilla.scss (hunk 10 above)
Comment (nottrobin) [reply _sass/_old_vanilla.scss]: You should be able to port over just that class. No need to bring the whole file over.

Comment (anthonydillon) [reply _sass/_old_vanilla.scss]: Oh no we should use the styling in the existing docs pages. For example: https://github.com/canonical-websites/www.ubuntu.com/blob/master/static/sass/_layout_documentation.scss

File: index.html [hunk 11]
```diff
@@ -1,290 +1,227 @@
-<!DOCTYPE html>
+---
```
Comment (anthonydillon) [inline index.html]: I think we should revert the changes to the homepage in this branch

File: README.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
-# microk8s.io
+# MicroK8s.io
```
Comment (nottrobin) [inline README.md]: I'm guessing this and the following line should probably be removed?

File: index.html (hunk 11 above)
Comment (nottrobin) [reply index.html]: What does this mean?

File: _sass/_old_vanilla.scss (hunk 10 above)
Comment (anthonydillon) [reply _sass/_old_vanilla.scss]: Thanks, looking good. Just need to add:
```css
.docs-container {
  max-width: 64.875rem;
  margin: 0 auto;
}
```

**Comment by Greggless** [comment]
+89345 🔥 🔥 🔥
